includes/Modules/Subscribe_With_Google.php

```diff
@@ -138,6 +164,14 @@ protected function setup_assets() {
 					),
 				)
 			),
+			new Script(
+				'googlesitekit-subscribe-with-google-bulk-edit',
+				array(
+					'src'           => $base_url . 'js/googlesitekit-subscribe-with-google-bulk-edit.js',
+					'dependencies'  => array(),
```

Comment:
You may need to specify a few minimum dependencies here, e.g. `googlesitekit-runtime` or `googlesitekit-vendor` - maybe that's what's missing here.

Reply:
The `googlesitekit-vendor` dependency fixes it, thank you!